public/locales/en/modules/dns-hole-summary.json

```diff
@@ -0,0 +1,20 @@
+{
+  "descriptor": {
+    "name": "DNS hole summary",
+    "description": "Displays important data from PiHole or AdGuard",
+    "settings": {
+      "title": "Settings for DNS Hole summary",
+      "usePiHoleColors": {
+        "label": "Use colors from PiHole"
+      }
+    }
+  },
+  "card": {
+    "metrics": {
+      "domainsOnAdlist": "Domains on adlists",
+      "queriesToday": "Queries today",
+      "queriesBlockedTodayPercentage": "blocked today",
```

Comment:
IMO it would make sense to include the percentage value in the translation so that if in other languages the percentage value is not on the front it would work.